### PR TITLE
Add: Search by app id

### PR DESCRIPTION
**Pull request by rmeena840**
Add: Search by app id
Task: https://www.pivotaltracker.com/story/show/182496793

### CODE REVIEW

File: src/api/query.request.ts [hunk 1]
```diff
@@ -121,3 +121,18 @@ export const lookupAccountAppLocalStates = (addr: string, appId: number) => {
     }
   }`;
 };
+
+/** LookUpSigmaDaoByAppId
+ * @param appId Application ID
+ */
+export const getSearchByAppIdReq = (appId: number) => {
+	return `query SearchByAppId {
```
Comment (Megha-Dev-19) [inline src/api/query.request.ts]: may be we can update it to `getDaoInfoByAppIdReq`

File: src/pages/AllDao.vue [hunk 2]
```diff
@@ -389,6 +395,19 @@ export default defineComponent({
 				}
 			}
 		},
+		async searchDaoBy(searchDaoType: SearchDaoType, appId: number) {
+			switch (searchDaoType) {
```
Comment (Megha-Dev-19) [inline src/pages/AllDao.vue]: similarly can we update it to `handleDaoSearch`

**Review by Megha-Dev-19 — APPROVED** [review]
Nice work!

File: src/api/query.request.ts (hunk 1 above)
Comment (rmeena840) [reply src/api/query.request.ts]: Done

File: src/pages/AllDao.vue (hunk 2 above)
Comment (rmeena840) [reply src/pages/AllDao.vue]: Done

**Pull request merged**